FWCore/Services/plugins/Timing.cc

```diff
@@ -310,6 +316,42 @@ namespace edm {
         iRegistry.watchPostSourceConstruction(this, &Timing::postModule);
       }
 
+      iRegistry.preESModuleSignal_.connect([this](auto const& key, auto const& context) {
```

Comment:
How about
```suggestion
      iRegistry.preESModuleSignal_.connect([this](auto const& recordKey, auto const& context) {
```
to make the purpose of the first argument more clear without having to look the `ActivityRegistry::preESModuleSignal_`?
(I was also thinking to suggest adding explicit types, but then concluded their use below is simple enough to be understood with `auto` types)